common/src/main/java/com/wynntils/utils/wynn/LocationUtils.java

```diff
@@ -12,10 +12,10 @@
 
 public final class LocationUtils {
     private static final Pattern COORDINATE_PATTERN = Pattern.compile(
-            "(?<x>[-+]?\\d{1,6})(?:\\.\\d+)?([^0-9.+-]{1,5}(?<y>[-+]?\\d{1,3})(?:\\.\\d+)?)?[^0-9.+-]{1,5}(?<z>[-+]?\\d{1,6})(?:\\.\\d+)?");
+            "\\s*(?<x>[-+]?\\d{1,6})(?:\\.\\d+)?([^0-9.+-]{1,5}(?<y>[-+]?\\d{1,3})(?:\\.\\d+)?)?[^0-9.+-]{1,5}(?<z>[-+]?\\d{1,6})(?:\\.\\d+)?\\s*");
 
-    private static final Pattern STRICT_COORDINATE_PATTERN = Pattern.compile(
-            "(?:^|\\s)([-+]?\\d{1,6})(?:\\.\\d+)?((,\\s?|\\s)([-+]?\\d{1,3})(?:\\.\\d+)?)?(,\\s?|\\s)([-+]?\\d{1,6})(?:\\.\\d+)?(?:\\s|$)");
+    private static final Pattern STRICT_COORDINATE_PATTERN =
+            Pattern.compile("(?:^|\\s)([-+]?\\d{1,6}(?:[\\s,]+[-+]?\\d{1,3}(?:[\\s,]+[-+]?\\d{1,6})?)?)(?:\\s+|$)");
```

Comment:
Perhaps limit the `[\\s,]` groups to a certain max count? Like 0-2 chars (inclusive)? This would match `100       ,    10 10` at the moment.

Reply:
I think Minecraft auto removes more than 1 whitespace, so theoretically you should never be able to send that

Reply:
Same with 1000,,,,,100 then